src/VisualStudio/Core/Def/Implementation/Interop/Telemetry.cs

```diff
@@ -21,6 +21,8 @@ internal interface IVsTelemetrySession
         void PostDoublePiiProperty([ComAliasName("OLE.LPCOLESTR")]string szPropertyName, double varValue);
```

Comment:
The method order for the interfaces defined in `Telemetry.cs` does not match the method order for the interfaces defined in the internal idl file.

I added the new methods inline with how `Telemetry.cs` was already layed-out (in alphabetical order with properties appearing before methods).

Should I modify and resubmit so that this exactly matches the `IDL` and native `VTBL`?